cds/modules/deposit/api.py

```diff
@@ -573,6 +575,33 @@ def _current_tasks_status(self):
         return get_tasks_status_by_task(
             get_deposit_events(self['_deposit']['id']))
 
+    @contextmanager
+    def _process_files(self, record_id, data):
```

Comment:
I would move this to the [CDSDeposit](https://github.com/CERNDocumentServer/cds/blob/cdslabs_qa/cds/modules/deposit/api.py#L124) class, we will use it in every deposit, i.e. pdf <- pdfa